OxfordBodleian/Juel-Jensen/BDLaethf33.xml

```diff
@@ -33,21 +36,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <idno>Juel-Jensen 43</idno>
                         </altIdentifier>
                     </msIdentifier>
-                    
+
                     <msContents>
                         <summary/>
-                        
+
                         <!--<msItem xml:id="ms_i1">
-                      
+
                         </msItem>-->
-                        
+
                     </msContents>
-                    
+
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p>Holes, presumably resulting from parchment making process, are found on <locus target="#1 #25"/>.
```

Comment:
Again, maybe "tears"?

Reply:
Same as before, they are rounded in shape.